packages/rum-core/src/performance-monitoring/performance-monitoring.js

```diff
@@ -217,7 +212,7 @@ class PerformanceMonitoring {
 
     if (!duration) {
       if (__DEV__) {
-        let message = 'Transaction was discarded! '
+        let message = `Transaction(${tr.name}) was discarded! `
```

Comment:
Maybe it should be the transaction id instead, WDYT?

Reply:
We could definitely log id. But wont it be hard for user to figure out what that means?

Reply:
Logs id and name